tests/s25Main/UI/testIngameWindow.cpp

```diff
@@ -228,4 +232,171 @@ BOOST_AUTO_TEST_CASE(SaveAndRestoreMinimized)
     }
 }
 
+namespace {
+void WindowPositioning_testOne(IngameWindow& wnd, const char* msg, const std::function<void()>& main,
+                               const std::function<void()>& minimize)
```

Comment:
Maybe the param names can be cleaner:
```suggestion
void WindowPositioning_testOne(IngameWindow& wnd, const char* context, const std::function<void()>& checkNormal,
                               const std::function<void()>& checkMinimized)
```

And with the current usage it might be even better to move the context out of this function, i.e. have `BOOST_TEST_CONTEXT` at the call site instead of passing it as a param. But up to you.

Reply:
I prefer keeping it in the function. Waiting for `apt.llvm.org/focal` to come back before I push.